pkg/compliance/report/table.go

```diff
@@ -30,9 +29,8 @@ func (tw TableWriter) Write(report *ComplianceReport) error {
 	switch tw.Report {
 	case allReport:
 		t := pkgReport.Writer{
-			Output:          tw.Output,
-			Severities:      tw.Severities,
-			ShowMessageOnce: &sync.Once{},
```

Comment:
Note to reviewers: this field has been removed because it has not been used elsewhere